cumulus_library/base_utils.py

```diff
@@ -54,6 +54,12 @@ def get_schema(config: StudyConfig, manifest: study_manifest.StudyManifest):
     return config.schema
 
 
+def get_prefix_with_seperator(manifest: study_manifest.StudyManifest):
+    if dedicated := manifest.get_dedicated_schema():
+        return f"{dedicated}."
+    return f"{manifest.get_study_prefix()}__"
```

Comment:
Should this just be a method on StudyManifest?

Reply:
i suppose - i started here because of the get_schema method, but without the config dependency that probably makes sense.